dts/arm/nordic/nrf52832.dtsi

```diff
@@ -44,6 +44,28 @@
 			status = "disabled";
 			label = "UART_0";
 		};
+
+		i2c0: i2c@40003000 {
+			compatible = "nordic,nrf5-i2c";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40003000 0x1000>;
+			interrupts = <3 1>;
+			default_cfg = <0x04>;
```

Comment:
We shouldn't have default_cfg, this driver should be updated to use _i2c_map_dt_bitrate with clock-frequency property in dts.  You can see commit 54933b3833 for an example.